src/Component-Routes/Feed/Feed.css

```diff
@@ -0,0 +1,165 @@
+.notesContainer{
```

Comment:
En css casi no se usa camel case
```suggestion
.notes-container{
```
en vez de eso podrías usar guiones -